ts/Core/Pointer.ts

```diff
@@ -1830,25 +1844,36 @@ class Pointer {
      * @private
      * @function Highcharts.Pointer#scaleGroups
      */
-    public scaleGroups(attribs?: Series.PlotBoxObject, clip?: boolean): void {
+    public scaleGroups(attribs?: Series.PlotBoxObject, clip?: BBoxObject, pinchedAxes?: Axis[]): void {
 
         const chart = this.chart;
-
+        let axes = pinchedAxes || chart.axes;
         // Scale each series
-        chart.series.forEach(function (series): void {
-            const seriesAttribs = attribs || series.getPlotBox(); // #1701
-            if (series.xAxis && series.xAxis.zoomEnabled && series.group) {
-                series.group.attr(seriesAttribs);
-                if (series.markerGroup) {
-                    series.markerGroup.attr(seriesAttribs);
-                    series.markerGroup.clip(
-                        clip ? (chart.clipRect as any) : (null as any)
-                    );
-                }
-                if (series.dataLabelsGroup) {
-                    series.dataLabelsGroup.attr(seriesAttribs);
+        axes.forEach((axis: Axis): void => {
+            axis.series.forEach((series: Series): void => {
+                const seriesAttribs = attribs || series.getPlotBox(); // #1701
+                const isX = axis.isXAxis;
```

Comment:
In the Highcharts project, we use "1 key word declaration pattern". Use one `const` instead of multiple. The same with `let` below